sdk/tests/js/tests/tex-input-validation.js

```diff
@@ -439,10 +439,14 @@ debug("");
 debug("Checking CopyTexSubImage2D: a set of inputs that are valid in GL but invalid in WebGL");
 
 testCases = [
+  // Section 3.8.5 of the GLES 3.0.3 spec, pg 140, requires that, if the
```

Comment:
Updated to 3.0.4, but it's actually pg 140 (starting there, but effective format is defined in 141)